holmes/clients/robusta_client.py

```diff
@@ -1,18 +1,20 @@
 import logging
 from typing import Optional
 import requests  # type: ignore
+from functools import cache
 from pydantic import BaseModel, ConfigDict
 from holmes.common.env_vars import ROBUSTA_API_ENDPOINT
 
 HOLMES_GET_INFO_URL = f"{ROBUSTA_API_ENDPOINT}/api/holmes/get_info"
-TIMEOUT = 2
+TIMEOUT = 0.3
 
 
 class HolmesInfo(BaseModel):
     model_config = ConfigDict(extra="ignore")
     latest_version: Optional[str] = None
 
 
+@cache
```

Comment:
Will this reduce the call to holmes across `holmes ask` commands?

Reply:
@mainred can you explain the question further?

Reply:
Normally we call fetch_holmes_info once for each `holmes ask`, so if the cache does not work other, following, `holmes ask`, it brings no benefit.

Reply:
@mainred it helps here because `model_post_init` is called twice in certain flows, so without this cache, `fetch_holmes_info` is also called twice which can be expensive.